test/NelsonSiegelSvensson.jl

```diff
@@ -0,0 +1,27 @@
+@testset "NelsonSiegelSvensson" begin
```

Comment:
Please also add some tests of the `zero` function, and ensure that some of the other primary functions like `discount`, `forward`, and `accumulation` work. I think they should since you've defined `zero` and the [fallback methods](https://github.com/JuliaActuary/Yields.jl/blob/master/src/generics.jl) but would be good to have them in the test suite. It would be best if you could validated from an external source what the values of those functions should be, but I think at least testing that they don't error (e.g. `@test forward(ns,1,2) isa Rate`)

Reply:
Will do.